wee_slack.py

```diff
@@ -683,7 +688,7 @@ def buffer_prnt(self, user='unknown_user', message='no message', time=0):
                 else:
                     name = user
                     self.last_active_user = None
-                name = nick_prefix + name + nick_suffix
+                name = nick_prefix_color + nick_prefix + name + nick_suffix_color + nick_suffix + w.color("reset")
```

Comment:
Doesn't this need a `w.color("reset")` before name in (arguably very weird) case you've disabled nick colouring, but use `weechat.color.chat_nick_prefix`?